appveyor.yml

```diff
@@ -1,8 +1,3 @@
-branches:
-  only:
-    - dev
```

Comment:
Commits won't be built unless they have an appveyor.yml file checked in, which for the moment is only on dev. We can re-add this if it becomes a problem for any reason, I think.